### ISSUE TITLE
Can't add new Layouts

### ISSUE THREAD

**Issue by agzam** [syl20bnr/spacemacs#9548]
#### Description :octocat:
Can't add new layouts

#### Reproduction guide :beetle:
- make sure `dotspacemacs-auto-generate-layout-names` set to `nil` 
- Start Emacs
- `SPC l 2`
- When prompted, press `y`
- Give any name for the layout, e.g.: `foo`

*Observed behaviour:* :eyes: :broken_heart:
- No new layout gets created
- `simpler-helm-bookmark-keybindings: Symbol’s value as variable is void: helm-bookmark-map` error is thrown

#### System Info :computer:
- OS: darwin
- Emacs: 25.2.1
- Spacemacs: 0.200.9
- Spacemacs branch: develop (rev. 6d73df70)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(csv yaml emacs-lisp lua javascript
     (haskell :variables haskell-process-type 'stack-ghci)
     (html :packages
           (not pug-mode slim-mode))
     (markdown :packages
               (not mmm-mode)
               :variables markdown-live-preview-engine 'vmd)
     (shell-scripts :packages
                    (not fish-mode))
     (auto-completion :variables auto-completion-tab-key-behavior 'cycle auto-completion-enable-snippets-in-popup t auto-completion-enable-help-tooltip t auto-completion-enable-sort-by-usage t)
     syntax-checking spell-checking helm fasd osx restclient emoji search-engine imenu-list docker pdf-tools
     (shell :variables shell-enable-smart-eshell t comint-scroll-show-maximum-output nil comint-move-point-for-output nil)
     git
     (version-control :variables version-control-diff-tool 'diff-hl version-control-global-margin t)
     (github :packages
             (not magit-gh-pulls)
             :variables gist-view-gist t magithub-api-timeout 5)
     ag-dired ag-general ag-synonyms ag-clojure ag-web
     (ag-org :variables org-enable-reveal-js-support t org-enable-bootstrap-support t org-enable-github-support t))
```
- System configuration features: JPEG RSVG IMAGEMAGICK NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

**Comment by agzam** [comment]
related https://github.com/syl20bnr/spacemacs/issues/9544

**Comment by agzam** [comment]
seems to be in relation to #9547, this issue may be closed once the PR is merged

**Comment by bmag** [comment]
I'm marking as duplicate of #9549 (I just saw it first), anyway should be fixed now.
